Zilon.Core/Zilon.Core/WorldGeneration/DirectBitmap.cs

```diff
@@ -49,7 +48,7 @@ public void Dispose()
             }
 
             Disposed = true;
-            Bitmap.Dispose();
+            //Bitmap.Dispose();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=4940399)